intercom_flutter_web/pubspec.yaml

```
@@ -24,4 +24,5 @@ dev_dependencies:
   mockito: ^5.4.0
 
 environment:
-  sdk: ">=3.0.0 <4.0.0"
\ No newline at end of file
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=2.0.0"
```

Why is this `flutter: ">=2.0.0"` line needed?

---

Just saw it - running `dart pub publish --dry-run` reports this error:
```
Package validation found the following error:
* pubspec.yaml allows Flutter SDK version 1.9.x, which does not support the flutter.plugin.platforms key.
  Please consider increasing the Flutter SDK requirement to ^1.10.0 (environment.sdk.flutter)
  
  See https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin
Sorry, your package is missing a requirement and can't be published yet.
For more information, see: https://dart.dev/tools/pub/cmd/pub-lish.
```

---

In this case, since Dart SDK 3 comes bundled with Flutter 3.10, I think this line should be:
```suggestion
  flutter: ">=3.10.0"
```